randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -188,9 +186,16 @@ Extra - asset_id: collision_camera_003
 > Center Platform; Heals? False
   * Layers: default
   > Dock to Teleport to Burenia (Cyan)
-      Grapple Beam or Gravity Suit or Use Spin Boost
+      Any of the following:
+          Flash Shift or Gravity Suit or Spider Magnet or Use Spin Boost
+          Grapple Beam and Grapple Movement (Beginner)
+          Lay Cross Comb and Perform WBJ
+  > Door to EMMI Zone Exit Middle (Power)
+      Trivial
   > Door to EMMI Zone Exit Middle (Charge)
-      Grapple Beam or Gravity Suit or Use Spin Boost
+      Any of the following:
+          Flash Shift or Gravity Suit or Spider Magnet or Lay Cross Comb or Use Spin Boost
+          Grapple Beam and Grapple Movement (Beginner)
```

Comment:
you forgot to do the "Perform WBJ" here, but im completely certain you can just IBJ normally from the center platform. I'm also sure a WBJ isnt even needed